public/index.html

```diff
@@ -13,8 +13,11 @@
 		gtag('js', new Date());
 
 		gtag('config', 'G-30VQJDBL2T');
-
 	</script>
+  	<script src="https://unpkg.com/vue"></script>
```

Comment:
Do we need this both in the package.json and here? I thought the bundler would take care of this.

Reply:
Agreed, the bundler should take of this. It seems like all the additions in index.html aren't necessary, but does it break without them @liangel02?